### DISCUSSION TITLE
Hard-coded IP

### DISCUSSION THREAD

**Discussion by morpig** [supabase/supabase#2287] [Questions] [answered]
hi,

currently trying out Supabase, and wondering if the "http://xxxxxxxxx.supabase.co" endpoint IP is dedicated per project? or is it a shared ip (with others too)?

can I hardcode those IPs? or it will change dynamically?

thanks!

**Comment by inian** [comment, marked as answer]
Hey @morpig, you cant rely on the IP being the same as it may change when we do upgrades and other maintenance for project
